cfme/tests/configure/test_default_views_cloud.py

```diff
@@ -33,84 +38,60 @@ def select_two_quads():
         fill(quad.checkbox(), True)
 
 
-@pytest.fixture(scope="module")
-def setup_a_provider():
-    _setup_a_provider(prov_class="cloud", prov_type="openstack", validate=True, check_existing=True)
-
-
-def set_view(group, button):
-    bg = ButtonGroup(group)
-    if bg.active != button:
-        bg.choose(button)
-        sel.click(form_buttons.save)
-
-
-def reset_default_view(name, default_view):
-    bg = ButtonGroup(name)
-    sel.force_navigate("my_settings_default_views")
-    if bg.active != default_view:
-        bg.choose(default_view)
-        sel.click(form_buttons.save)
-
-
-def get_default_view(name):
-    bg = ButtonGroup(name)
-    pytest.sel.force_navigate("my_settings_default_views")
-    default_view = bg.active
-    return default_view
-
-
 def set_and_test_default_view(group_name, view, page):
-    default_view = get_default_view(group_name)
-    set_view(group_name, view)
+    old_default = DefaultView.get_default_view(group_name)
+    DefaultView.set_default_view(group_name, view)
     if isinstance(page, basestring):
         sel.force_navigate(page)
     else:
         navigate_to(page, 'All', use_resetter=False)
     assert tb.is_active(view), "{} view setting failed".format(view)
-    reset_default_view(group_name, default_view)
+    DefaultView.set_default_view(group_name, old_default)
+
+# BZ 1283118 written against 5.5 has a mix of what default views do and don't work on different
+# pages in different releases
 
 
 @pytest.mark.parametrize('key', gtl_params, scope="module")
-def test_tile_defaultview(request, setup_a_provider, key):
```

Comment:
Should be covered by line 14, `pytest.mark.usefixtures('setup_a_provider')`. Added it there since every function uses it.